build/BUILD

```diff
@@ -1,7 +1,7 @@
 package(default_visibility = ["//visibility:public"])
 
 load("@io_bazel_rules_docker//container:container.bzl", "container_bundle", "container_image")
-load("@io_kubernetes_build//defs:build.bzl", "release_filegroup")
+load("@io_k8s_repo_infra//defs:build.bzl", "release_filegroup")
```

Comment:
Can we put this first commit in its own PR?

Reply:
I kept this in the same PR since I'm iterating on repo_infra. I guess I can bump to the latest merged repo-infra in a separate PR (along with docker updates etc) and then bump here again once my other PR merges.

Reply:
https://github.com/kubernetes/kubernetes/pull/73983